src/drivers/imu/mpu9250/mpu9250_main.cpp

```diff
@@ -372,7 +280,7 @@ info(enum MPU9250_BUS busid)
 void
 usage()
 {
-	PX4_INFO("missing command: try 'start', 'info', 'test', 'stop',\n'reset', 'regdump', 'testerror'");
+	PX4_INFO("missing command: try 'start', 'info', 'test', 'stop',\n 'regdump', 'testerror'");
```

Comment:
We desperately need to standardize the list here.... it happens everywhere in the drivers and there is no standard.

Reply:
I don't think we should be worrying about standardizing a list, I think it should be gutted entirely and replaced with a single launcher/frontend.